lib/verifier.rb

```diff
@@ -94,4 +94,20 @@ def verify_file filename
     verify manifest
   end
 
+  def verify_topics filename
+    manifest = Manifest.parse_file filename
+
+    print "Verify topics of " + manifest.filename + "..."    
+    topics_list =  manifest.topics
+    if topics_list.nil?
+      puts "Warning: Specify the topics"        
+    else
+      if !topics_list.kind_of?(Array) 
+        puts "Error: Topics are not in an array"  
```

Comment:
This will print an error on stdout, but not make the validation fail. The validation only fails, if errors are recorded in the result object. See how other methods are doing it.